man/asciidoctor.1

```diff
@@ -173,43 +173,47 @@ Matching templates found in subsequent directories override ones previously disc
 \fB\-\-failure\-level\fP=\fILEVEL\fP
 .RS 4
 The minimum logging level that triggers a non\-zero exit code (failure).
-If this option is not set (default: FATAL), the program exits with a status code zero even if warnings or errors have been logged.
+If this option is not set (default: FATAL), the program exits with exit code zero even if warnings or errors have been logged.
 .RE
 .sp
 \fB\-q, \-\-quiet\fP
 .RS 4
-Silence warnings.
+Silence application log messages and script warnings.
 .RE
 .sp
 \fB\-\-trace\fP
 .RS 4
-Include backtrace information on errors.
-Not enabled by default.
+Include backtrace information when reporting errors.
 .RE
 .sp
 \fB\-v, \-\-verbose\fP
 .RS 4
-Verbosely print processing information and configuration file checks to stderr.
+Verbosely print processing information to stderr, including debug\-level log messages.
+.RE
+.sp
+\fB\-w, \-\-warnings\fP
+.RS 4
+Turn on script warnings (applies to executed code).
 .RE
 .sp
 \fB\-t, \-\-timings\fP
 .RS 4
-Display timings information (time to read, parse and convert).
+Print timings report to stderr (time to read, parse and convert).
```

Comment:
You mean `stdout` ? https://github.com/asciidoctor/asciidoctor/blob/eb251b6e704396219f3cb184396cc2ec1a641e6f/lib/asciidoctor/timings.rb#L49

Reply:
Nope, the CLI sends the report to stderr (so it doesn't get mixed up with the converted document going to stdout).

Reply:
Just checked, it's `stderr`